chocolatey/boxstarter.org
# (#52) Update gulp dependencies

## Description Of Changes
Dependencies that are used in gulpfile.js have been updated to use the recommended packages. For logging errors, choco-theme was relying on a depreciated package `gulp-util`. This package has been replaced with the recommended package `fancy-log`.

## Motivation and Context
The gulp-util package was causing dependabot security alerts.

## Testing
No UI changes were made in choco-theme that will effect this site.

1. Pull this PR down
2. Ensure the site builds locally
3. If everything succeeds then it is good 

### Operating Systems Testing
n/a

## Change Types Made


## Change Checklist


## Related Issue
* ENGTASKS-2920
* #52
* This choco-theme PR: https://github.com/chocolatey/choco-theme/pull/319
